Runtime/Scripts/LyumaAv3Runtime.cs

```diff
@@ -2100,6 +2295,71 @@ void NormalUpdate()
 						defaultHeadScale = head.localScale;
 					}
 					head.localScale = EnableHeadScaling ? new Vector3(0.0001f, 0.0001f, 0.0001f) : defaultHeadScale; // head bone is set to 0.0001 locally (not multiplied
+
+					Type headChopType = Type.GetType("VRC.SDK3.Avatars.Components.VRCHeadChop, VRCSDK3A, Version=1.0.0.0, Culture=neutral, PublicKeyToken=null");
+					if (headChopType != null)
+					{
+						Vector3 div(Vector3 x, Vector3 y) => new Vector3(x.x / y.x, x.y / y.y, x.z / y.z);
+						Vector3 mul(Vector3 x, Vector3 y) => new Vector3(x.x * y.x, x.y * y.y, x.z * y.z);
+						if (headChopData == null)
+						{
+							headChops = GetComponentsInChildren(headChopType);
+							headChopData = new Dictionary<Transform, HeadChopDataStorage>();
+							for (var i = 0; i < headChops.Length; i++)
+							{
+								dynamic headChop = headChops[i];
+								object[] bones = headChop.targetBones;
+								for (var j = 0; j < bones.Length; j++)
+								{
+									dynamic bone = bones[j];
+									Transform t = bone.transform;
+									headChopData[t] = new HeadChopDataStorage()
+									{
+										originalLocalPosition = t.localPosition,
+										originalRootSpacePosition = mul(avadesc.transform.InverseTransformPoint(t.position), avadesc.transform.lossyScale),
+										originalLocalScale = t.localScale,
+										originalGlobalScale = t.lossyScale
+									};
+								}
+							}
+						}
+						
+						for (var i = 0; i < headChops.Length; i++)
+						{
+							dynamic headChop = headChops[i];
```

Comment:
```suggestion
							dynamic headChop = headChops[i];
							bool isActiveAndEnabled = headChop.isActiveAndEnabled;
```

Cache for performance but I didn't benchmark.